### PR TITLE
Add AddNewTile component

### PR DESCRIPTION
**Pull request by mateuszkula**
I want to merge this change because...

It adds custom tile for adding new addresses/credit cards etc

### CODE REVIEW

File: src/@next/components/atoms/AddNewTile/test.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { mount } from "enzyme";
```
Comment (jwm0) [inline src/@next/components/atoms/AddNewTile/test.tsx]: I believe shallow should be sufficient for this use case. `shallow` is faster than `mount`

File: src/@next/components/atoms/AddNewTile/test.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import "jest-styled-components";
+import React from "react";
+
+import { AddNewTile } from ".";
+
+describe("<AddNewTile />", () => {
+  // Example test
```
Comment (jwm0) [inline src/@next/components/atoms/AddNewTile/test.tsx]: Remove comment and make the test name more explicit like "renders correct caption based on provided prop"

Comment (mateuszkula) [reply src/@next/components/atoms/AddNewTile/test.tsx]: Fixed 🌮

File: src/@next/components/atoms/AddNewTile/test.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { mount } from "enzyme";
+import "jest-styled-components";
+import React from "react";
+
+import { AddNewTile } from ".";
+
+describe("<AddNewTile />", () => {
+  it("AddNewTile component exists and proper text content is passed to Tile component", () => {
```
Comment (AlicjaSzu) [inline src/@next/components/atoms/AddNewTile/test.tsx]: In my opinion it("should exists......") is better

File: src/@next/components/atoms/Tile/test.tsx [hunk 4]
```diff
@@ -54,4 +54,38 @@ describe("<Tile />", () => {
...
+    );
+    expect(wrapperWithAddNew).toHaveStyleRule(
+      "color",
+      defaultTheme.colors.white,
+      {
+        modifier: ":hover",
+      }
+    );
+  });
 });
```
Comment (AlicjaSzu) [inline src/@next/components/atoms/Tile/test.tsx]: maybe you should add tests for conditionally rendered header and footer?

Comment (mateuszkula) [reply src/@next/components/atoms/Tile/test.tsx]: I have added tests for conditional rendering

File: src/@next/components/atoms/AddNewTile/test.tsx (hunk 3 above)
Comment (mateuszkula) [reply src/@next/components/atoms/AddNewTile/test.tsx]: Fixed

File: src/@next/components/atoms/Tile/test.tsx [hunk 5]
```diff
@@ -55,6 +56,29 @@ describe("<Tile />", () => {
     );
   });
 
+  it("should display only element if no footer and header elements passed", () => {
```
Comment (AlicjaSzu) [inline src/@next/components/atoms/Tile/test.tsx]: only one element?

Comment (mateuszkula) [reply src/@next/components/atoms/Tile/test.tsx]: Yes :) it should be only one element. Fixed

**Pull request merged**